### ISSUE TITLE
Noisy "DirectWrite: CreateFontFaceFromHDC() failed" errors in console

### ISSUE THREAD

**Issue by maphew** [leo-editor/leo-editor#1262]
The error messages to shell window when a font specified in myLeoSettings.leo isn't found is very noisy. Secondly, there's no corresponding message to the log pane.

```
> leo-messages

Leo 6.0-b2-devel, devel branch, build 5774f23cae
2019-07-23 17:18:21 -0500
DirectWrite: CreateFontFaceFromHDC() failed (Unspecified error) for QFontDef(Family="DejaVu Sans Mono", pointsize=10, pixelsize=13, styleHint=5, weight=50, stretch=100, hintingPreference=0) LOGFONT("DejaVu Sans Mono", lfWidth=0, lfHeight=-13) dpi=96
QWindowsFontEngine::QWindowsFontEngine: GetTextMetrics failed (The operation completed successfully.)
DirectWrite: CreateFontFaceFromHDC() failed (Unspecified error) for QFontDef(Family="DejaVu Sans Mono", pointsize=9.75, pixelsize=13, styleHint=5, weight=50, stretch=100, hintingPreference=0) LOGFONT("DejaVu Sans Mono", lfWidth=0, lfHeight=-13) dpi=96
QWindowsFontEngine::QWindowsFontEngine: GetTextMetrics failed (The operation completed successfully.)
DirectWrite: CreateFontFaceFromHDC() failed (Unspecified error) for QFontDef(Family="DejaVu Sans Mono", pointsize=8, pixelsize=11, styleHint=5, weight=50, stretch=100, hintingPreference=0) LOGFONT("DejaVu Sans Mono", lfWidth=0, lfHeight=-11) dpi=96
QWindowsFontEngine::QWindowsFontEngine: GetTextMetrics failed (The operation completed successfully.)
DirectWrite: CreateFontFaceFromHDC() failed (Unspecified error) for QFontDef(Family="DejaVu Sans Mono", pointsize=8.25, pixelsize=11, styleHint=5, weight=50, stretch=100, hintingPreference=0) LOGFONT("DejaVu Sans Mono", lfWidth=0, lfHeight=-11) dpi=96
QWindowsFontEngine::QWindowsFontEngine: GetTextMetrics failed (The operation completed successfully.)
DirectWrite: CreateFontFaceFromHDC() failed (Unspecified error) for QFontDef(Family="DejaVu Sans Mono", pointsize=9, pixelsize=12, styleHint=5, weight=50, stretch=100, hintingPreference=0) LOGFONT("DejaVu Sans Mono", lfWidth=0, lfHeight=-12) dpi=96
QWindowsFontEngine::QWindowsFontEngine: GetTextMetrics failed (The operation completed successfully.)
DirectWrite: CreateFontFaceFromHDC() failed (Unspecified error) for QFontDef(Family="DejaVu Sans Mono", pointsize=9, pixelsize=12, styleHint=5, weight=50, stretch=100, hintingPreference=0) LOGFONT("DejaVu Sans Mono", lfWidth=0, lfHeight=-12) dpi=96
QWindowsFontEngine::QWindowsFontEngine: GetTextMetrics failed (The operation completed successfully.)
DirectWrite: CreateFontFaceFromHDC() failed (Unspecified error) for QFontDef(Family="DejaVu Sans Mono", pointsize=11, pixelsize=15, styleHint=5, weight=50, stretch=100, hintingPreference=0) LOGFONT("DejaVu Sans Mono", lfWidth=0, lfHeight=-15) dpi=96
QWindowsFontEngine::QWindowsFontEngine: GetTextMetrics failed (The operation completed successfully.)
DirectWrite: CreateFontFaceFromHDC() failed (Unspecified error) for QFontDef(Family="DejaVu Sans Mono", pointsize=11.25, pixelsize=15, styleHint=5, weight=50, stretch=100, hintingPreference=0) LOGFONT("DejaVu Sans Mono", lfWidth=0, lfHeight=-15) dpi=96
QWindowsFontEngine::QWindowsFontEngine: GetTextMetrics failed (The operation completed successfully.)
```

**Comment by maphew** [comment]
The error didn't go away after I installed DejaVu fonts (though I haven't rebooted, trying that next).

https://bugreports.qt.io/browse/QTBUG-57180 indicates this is an upstream error when a font incorrectly reports it's scaling ability. Present for a long time but Qt is only emitting messages about it now.

**Comment by maphew** [comment]
I decided to try `pip install --upgrade pyqt5` first, which upgraded me from v5.12 to v5.13.0 The noise is better but not absent. Closing the issue, but this might indicate a need to up the minimum pyqt level. Waiting to see if more problems are reported first.

~~~
> leo-m

Leo 6.0-b2-devel, devel branch, build 5774f23cae
2019-07-23 17:18:21 -0500
QWindowsFontEngine::QWindowsFontEngine: GetTextMetrics failed (The operation completed successfully.)
QWindowsFontEngine::QWindowsFontEngine: GetTextMetrics failed (The operation completed successfully.)
QWindowsFontEngine::QWindowsFontEngine: GetTextMetrics failed (The operation completed successfully.)
QWindowsFontEngine::QWindowsFontEngine: GetTextMetrics failed (The operation completed successfully.)
QWindowsFontEngine::QWindowsFontEngine: GetTextMetrics failed (The operation completed successfully.)
QWindowsFontEngine::QWindowsFontEngine: GetTextMetrics failed (The operation completed successfully.)
QWindowsFontEngine::QWindowsFontEngine: GetTextMetrics failed (The operation completed successfully.)
QWindowsFontEngine::QWindowsFontEngine: GetTextMetrics failed (The operation completed successfully.)
wrote D:/Matt/Dropbox/.leo/leo.session
wrote recent file: D:/Matt/Dropbox/.leo/.leoRecentFiles.txt
~~~

~~~
Leo 6.0-b2-devel, devel branch, build 5774f23cae
2019-07-23 17:18:21 -0500
Python 3.6.3, PyQt version 5.13.0
Windows 10 AMD64 (build 10.0.17763) SP0
~~~
